lib/gyro/template.rb

```diff
@@ -54,24 +72,14 @@ def self.directory
     #         A 2-items array of [the alias name, the alias target name]
     #         Or nil if the entry does not correspond to a valid template alias
     #
-    def self.resolve_alias(entry)
-      return nil unless entry.exist?
-      return nil if entry.extname != '.alias'
-      base = entry.basename('.alias').to_s
-      target = entry.open(&:readline).chomp # Only read the first line of the file, the rest is ignored
-      return nil if File.basename(target) != target || target.include?('..') # Security measure
-      [base, target]
-    end
+    def self.resolve_alias(name)
+      template_config_path = Gyro::Template.directory + 'config.yml'
+      config = YAML.load_file(template_config_path)
+      base = name
+      target = config['alias'][name]
 
-    # Resolve alias by name
-    #
-    # @return [(String, String)]
-    #         A 2-items array of [the alias name, the alias target name]
-    #         Or nil if the name does not correspond to a valid template alias
-    #
-    def self.resolve_alias_name(name)
-      path = Gyro::Template.directory + (name + '.alias')
-      resolve_alias(path)
+      return nil unless target
+      [base, target]
```

Comment:
Since now `base` == `name` (we don't have an extension to remove from the name), it's probably not necessary anymore to return `[base, target]`, you can just return `target` if it's indeed an alias in the `config.yml`, and `nil` if it isn't.

Reply:
Yes you are right, I just make a mistake when I refact this method.